src/components/resume/styles.ts

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const IfOpenTrue = styled.div<{open: boolean}>`
+    display: ${(props) => (props.open ? 'block':'none')}
+    `;
+
+export const StyledLayout = styled.span`
+    display: inline-block;
+    text-decoration: underline;
+    text-decoration: overline;
+    cursor: pointer
+    `;
```

Comment:
can we fix spacing and newline here?

Reply:
Corrected indents and added new line to the bottom